server/apps/main/templates/main/share_experiences.html

```diff
@@ -0,0 +1,141 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
+        integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+    <title>Share Experiences</title>
+    <style>
+        body {
+            background: #E8F3FC;
+        }
+        .btn-colour{
+            background: #A0F832;
+            height: 70px;
+            width: 150px;
+        
+        }
+        .btn-colour:hover{
+            background: #7dce1b;
+            height: 70px;
+            width: 150px;
+        
+        }
+        .navbar{
+            background: rgba(149, 239, 234, 0.42);
+            
+        }
+        .form-control{
+           width: 50%;
+
+        }
+        form{
+          padding: 3%;
+        }
+        .form-check-label{
+          font-size: 22px;
+          margin-left: 10px;
+        }
+        input.form-check-input {
+          width: 20px;
+          height: 20px;
+        }
+        textarea.form-control{
+          background-color: rgb(177, 200, 243);
+          color: #000;
+        }
+        textarea.form-control:focus{
+          background-color: rgb(177, 200, 243);
+          border-color: rgba(43, 117, 0, 0.6);
+          box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075), 0 0 8px rgba(43, 117, 0, 0.6);
+          outline: 0 none;
+          color: #000;
+        }
+        
+
+    </style>
+</head>
+
+<body>
+    <nav class="navbar navbar-expand">
+        <a class="navbar-brand" href="#">
+          <img src="autistic.png" width="65" height="35" alt="">
+          <img src="alanturing.png" width="75" height="35" alt="">
+        </a>
+            <ul class="navbar-nav ">
+              <li class="nav-item">
+                <a class="nav-link text-dark" href="{% url 'main:overview' %}">Home</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link text-dark" href="#">Share</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link text-dark" href="{% url 'main:list' %}">My Stories</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link text-dark" href="#">Logout</a>
```

Comment:
Just need to add the missing links per the ticket #16 and this is good to merge!

Reply:
Yes sure! I've pushed the new changes now.

Reply:
Well done!